### PR TITLE
Re-write Maif connector 

### PR DESCRIPTION
**Pull request by jacquarg**
Improve robustness (all connection data in the konnector);
Better error messages.

### CODE REVIEW

**Comment by poupotte** [comment]
It works for me :)

File: server/konnectors/maif.js [hunk 1]
```diff
@@ -30,234 +32,177 @@ if (nonce === '') {
...
-        moment.locale('fr');
-        const importDate = moment().format('LLL');
-        const payload = { profile: JSON.parse(body), date: importDate };
-
-        maifuser.updateAttributes(payload, (err) => { // mise à jour du maifuser en base en insérant le token
-          callback(err);
-        });
-      }
-    }, false);
+  Konnector.all((err, konnectors) => {
```
Comment (ZeHiro) [inline server/konnectors/maif.js]: Can you use the bySlug request?

File: server/models/requests.coffee [hunk 2]
```diff
@@ -44,3 +44,5 @@ module.exports =
 
     track:
         all: cozydb.defaultRequests.all
+
+    maifuser: all: cozydb.defaultRequests.all
```
Comment (ZeHiro) [inline server/models/requests.coffee]: can you put the request on a new line please .

**Comment by ZeHiro** [comment]
@jacquarg Cannot merge, please rebase 😄

**Pull request merged**